be/src/exprs/in-predicate-benchmark.cc

```diff
@@ -158,6 +158,7 @@
 #include <gutil/strings/substitute.h>
 
 #include "exprs/in-predicate.h"
+#include "exprs/in-predicate-ir.cc"
```

Comment:
This is the actual fix for the compile error in release mode. It looks to me like `in-predicate-ir.cc` is in the Makefiles generated for Release builds, so I'm not sure why the linker couldn't find it.

See [ROCANA-3468](https://scalingdata.atlassian.net/browse/ROCANA-3468) for the compile error.